app/models/user.rb

```diff
@@ -235,6 +235,17 @@ def grouped_recent_courses(number_of_years)
     subscribed_courses.group_by(&:year).first(number_of_years)
   end
 
+  def drawer_courses
+    actual_memberships = course_memberships.includes(:course).to_a.select(&:subscribed?)
+    favorites = actual_memberships.select(&:favorite)
+
+    return [] if actual_memberships.empty?
```

Comment:
```suggestion
    actual_memberships = course_memberships.includes(:course).to_a.select(&:subscribed?)
    return [] if actual_memberships.empty?

    favorites = actual_memberships.select(&:favorite)
```
For some reason, this looks a little better to me.